internal/interop/interopconnect/test_cases.go

```diff
@@ -657,6 +657,39 @@ func DoFailServerStreamingWithNonASCIIError(t crosstesting.TB, client connectpb.
 	t.Successf("successful fail server streaming with non-ASCII error")
 }
 
+func DoFailServerStreamingAfterResponse(t crosstesting.TB, client connectpb.TestServiceClient) {
+	respParam := make([]*testpb.ResponseParameters, len(respSizes))
+	for i, s := range respSizes {
+		respParam[i] = &testpb.ResponseParameters{
+			Size: int32(s),
+		}
+	}
+	req := &testpb.StreamingOutputCallRequest{
+		ResponseType:       testpb.PayloadType_COMPRESSABLE,
+		ResponseParameters: respParam,
+	}
+	stream, err := client.FailStreamingOutputCall(context.Background(), connect.NewRequest(req))
+	require.NoError(t, err)
+	for i := 0; i < 4; i++ {
```

Comment:
What do you think about doing:

```suggestion
	for i := 0; i < len(respSizes); i++ {
```

? We are asking the server for that many responses, so it seems fair to use exactly the same source of truth to assert the actual number of responses we receive.